### ISSUE TITLE
NVDA does not announce the position and number of tab items in a tab list (JTabbedPane, Java Access Bridge API)

### ISSUE THREAD

**Issue by anOolamr** [nvaccess/nvda#13896]
When navigating with the keyboard in a JTabbedPane tab list, NVDA does not announce the position and number of the currently focused tab item.

### Steps to reproduce:
1. If not available, download the SwingSet2 Demonstration Program from: https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/plaf.html#swingset2

2. Start NVDA.

3. Launch SwingSet2.jar with an up-to-date Java JRE.
-> The SwingSet2 application window appears.

4. Navigate with arrow right key to the "JButton demo" radio button and press spacebar to select it.

5. Press Tab key twice to navigate to the second JTabbedPane control, which has three tab items: Buttons, Radio Buttons, and Check Boxes.
-> NVDA announces "Buttons, tab, selected".
-> NVDA does not announce the tab item's position and the number of tab items.

6. Press arrow right/left keys or ctrl + Page-down/up shortcut to navigate to the other tab items.
-> NVDA announces the name, role and selected state of the tab items, but not their position and the number of tab items.

### Actual behavior:
NVDA announces the name, role and selected state of the tab items, but not their position and the number of tab items, although in NVDA settings the "Object Presentation" > "Report object position information" is marked.

### Expected behavior:
NVDA announces the name, role and selected state of the tab items and in addition their position and the number of tab items.

#### NVDA installed/portable/running from source:
installed.

#### NVDA version:
Version 2022.1

#### Windows version:
Version 10.0.19043 Build 19043

#### Name and version of other software in use when reproducing the issue:
java version "18.0.1" 2022-04-19
Java(TM) SE Runtime Environment (build 18.0.1+10-24)
Java HotSpot(TM) 64-Bit Server VM (build 18.0.1+10-24, mixed mode, sharing)

#### Other information about your system:
Java Access Bridge is enabled.

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, same behavior.

#### If NVDA add-ons are disabled, is your problem still occurring?
Not applicable.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable.

**Comment by mwhapples** [comment]
I seem to recall fixing this issue in a recent pullrequest #13744 so could you either try this with the build from the pullrequest or from master.

**Comment by seanbudd** [comment]
This should be fixed in 2022.2rc1
